zebra/zebra_nb_state.c

```diff
@@ -840,6 +840,58 @@ lib_vrf_zebra_ribs_rib_route_route_entry_nexthop_group_nexthop_color_get_elem(
 	return NULL;
 }
 
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:zebra/ribs/rib/route/route-entry/nexthop-group/nexthop/srv6-segs-stack/entry
+ */
+const void *
+lib_vrf_zebra_ribs_rib_route_route_entry_nexthop_group_nexthop_srv6_segs_stack_entry_get_next(
+	struct nb_cb_get_next_args *args)
+{
+	/* TODO: implement me. */
```

Comment:
Why are we not implementing this YANG support? I strongly encourage you to implement the support for dumping the SRV6 SIDs in YANG over-state

Reply:
After the new push, the next commit includes support for dumping the SRV6 SIDs in YANG oper-state: [commit](https://github.com/FRRouting/frr/pull/14089/commits/fb2accdb9407b4b00d05a7a9a86041e118897541)

Reply:
Example: 
`(config)# ipv6 route 2001:db8:1::/64 ens3 segments 2001:db8:2::/2001:db8:3::/2001:db8:4::
(config)# do show ipv6 route
Codes: K - kernel route, C - connected, S - static, R - RIPng,
       O - OSPFv3, I - IS-IS, B - BGP, N - NHRP, T - Table,
       v - VNC, V - VNC-Direct, A - Babel, D - SHARP, F - PBR,
       f - OpenFabric,
       > - selected route, * - FIB route, q - queued, r - rejected, b - backup
       t - trapped, o - offload failure

K>* ::/0 [0/1024] via fe80::2, ens4, 00:02:14
S>* 1::/64 [1/0] unreachable (blackhole), weight 1, 00:01:42
S>* 2001:db8:1::/64 [1/0] is directly connected, ens3, seg6 2001:db8:2::,2001:db8:3::,2001:db8:4::, weight 1, 00:00:04
C * fe80::/64 is directly connected, ens4, 00:02:14
C>* fe80::/64 is directly connected, ens3, 00:02:14
C>* fec0::/64 is directly connected, ens4, 00:02:14`